libfwupd/fwupd-remote.c

```diff
@@ -443,6 +451,88 @@ fwupd_remote_set_password(FwupdRemote *self, const gchar *password)
 	priv->password = g_strdup(password);
 }
 
+/**
+ * fwupd_remote_set_auth_type:
+ * @self: a #FwupdRemote
+ * @auth_type: a #FwupdRemoteAuthType, e.g. %FWUPD_REMOTE_AUTH_TYPE_S3
+ *
+ * Sets the authentication type of the remote.
+ *
+ * Since: 2.0.3
```

Comment:
2.0.14 was the most recently released version, this should be 2.0.15 for newly introduced symbols.